dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/Objects.java

```diff
@@ -81,6 +82,7 @@
     USER( "user", User.class ),
     USERGROUP( "userGroup", UserGroup.class ),
     VISUALIZATION( "visualization", Visualization.class ),
+    EVENTVISUALIZATION( "eventVisualization", EventVisualization.class ),
```

Comment:
`EVENT_VISUALIZATION` ?

Reply:
This would be my preference but all other objects above follow the pattern `EVENTVISUALIZATION`. That's why I used the same pattern. I did not check if there is some code that might uppercase `EventVisualization` and try to find the respective enum `EVENTVISUALIZATION`...

Do you think is safe to switch to `EVENT_VISUALIZATION`?

Reply:
This would be my preference but all other objects above follow the pattern `EVENTVISUALIZATION`. That's why I used the same pattern. I did not check if there is some code that might uppercase `EventVisualization` and try to find the respective enum `EVENTVISUALIZATION`...

Do you think is safe to switch to `EVENT_VISUALIZATION`?